data/ka'het/ka'het.txt

```diff
@@ -10,21 +10,34 @@
 
 # Ship names:
 phrase "ka'het"
+	phrase
+		"builder beginning"
 	word
-		"Pae"
-		"An"
-		"Kai"
-		"Rea"
-		"So"
-		"Axi"
-		"Rela"
-		"Mar"
-		"Ci"
-		"Tia"
-		"Gat"
-		"Geto"
-		"Roa"
-		"Aar"
+		"'het"
+
+phrase "ka'het"
+	word
+		"Osaeli"
+		"Alix"
+		"Retilie"
+		"Veliante"
+		"Eavine"
+		"Celeatis"
+		"Nona"
+		"Palarei"
+		"Ceilia"
+		"Aerite"
+		"Letis"
+		"Maiar"
+		"Lateria"
+		"Gertima"
+		"Santari"
+		"Antamesa"
+		"Delia"
+	word
+		" "
```

Comment:
I don't think that this should be a space, as "ka'het syllable" words don't start capitalized. An apostrophe would be the obvious, boring replacement, so perhaps a hyphen `-` instead, or no spacer character at all.

Reply:
done